mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature

```diff
@@ -707,7 +708,7 @@ Feature: Loans tests
 
     # post owner, manual charge and payment method
     * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostOwner')
-    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostManualCharge')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostManualCharge') { extManualChargeId: #(extManualChargeId) }
```

Comment:
I investigated error logs more deeply and found out the truly reason of fail, so besides changes you have done please replace couple of lines
![image](https://user-images.githubusercontent.com/48057481/147760828-7f0000a8-522f-4e6a-9591-7cb9b39f06d8.png)
into 
![image](https://user-images.githubusercontent.com/48057481/147760864-d3b0331c-a811-4a3b-882e-cfae21eed07c.png)

Reply:
fixed

Reply:
please replace "And" after test into asterix "*" you can see what I mean on screenshot

Reply:
fixed